modules/grpc/otel/otel-protobuf-parser.cpp

```diff
@@ -286,3 +287,485 @@ syslogng::grpc::otel::protobuf_parser::parse(LogMessage *msg, const LogRecord &l
   /* .otel.log.span_id */
   _set_value(msg, ".otel.log.span_id", log_record.span_id(), LM_VT_BYTES);
 }
+
+static void
+_add_repeated_Exemplar_fields_with_prefix(LogMessage *msg, std::string &key_buffer, size_t key_prefix_length,
+                                          const char *key, RepeatedPtrField<Exemplar> exemplars)
+{
+  key_buffer.resize(key_prefix_length);
+  key_buffer.append(key);
+  key_buffer.append(".");
+  size_t length_with_dot = key_buffer.length();
+  char number_buf[G_ASCII_DTOSTR_BUF_SIZE];
+
+  uint64_t idx = 0;
+  for (const Exemplar &exemplar : exemplars)
+    {
+      key_buffer.resize(length_with_dot);
+      std::snprintf(number_buf, G_N_ELEMENTS(number_buf), "%" PRIu64, idx);
+      key_buffer.append(number_buf);
+      key_buffer.append(".");
+      size_t length_with_idx = key_buffer.length();
+
+      /* <prefix>.<key>.<idx>.filtered_attributes.<...> */
+      _add_repeated_KeyValue_fields_with_prefix(msg, key_buffer, length_with_idx, "filtered_attributes",
+                                                exemplar.filtered_attributes());
+
+      /* <prefix>.<key>.<idx>.time_unix_nano */
+      std::snprintf(number_buf, G_N_ELEMENTS(number_buf), "%" PRIu64, exemplar.time_unix_nano());
+      _set_value_with_prefix(msg, key_buffer, length_with_idx, "time_unix_nano", number_buf, LM_VT_INTEGER);
+
+      /* <prefix>.<key>.<idx>.value */
```

Comment:
`filtered_attributes` seems to be missing.

Reply:
I think it is a couple of lines above:

```cpp
      /* <prefix>.<key>.<idx>.filtered_attributes.<...> */
      _add_repeated_KeyValue_fields_with_prefix(msg, key_buffer, length_with_idx, "filtered_attributes",
                                                exemplar.filtered_attributes());
```